core/invoker/src/main/scala/whisk/core/containerpool/docker/RuncClient.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2015-2016 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package whisk.core.containerpool.docker
+
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext
+import scala.util.Failure
+import whisk.common.TransactionId
+import scala.util.Success
+import whisk.common.LoggingMarkers
+import whisk.common.Logging
+import akka.event.Logging.ErrorLevel
+
+/**
+ * Serves as interface to the docker CLI tool.
+ *
+ * Be cautious with the ExecutionContext passed to this, as the
+ * calls to the CLI are blocking.
+ *
+ * You only need one instance (and you shouldn't get more).
+ */
+class RuncClient(executionContext: ExecutionContext)(implicit log: Logging) extends RuncApi with ProcessRunner {
+    implicit private val ec = executionContext
+
+    // Determines how to run docker. Failure to find a Docker binary implies
+    // a failure to initialize this instance of DockerClient.
+    protected val runcCmd: Seq[String] = Seq("/usr/bin/docker-runc")
+
+    def pause(id: ContainerId)(implicit transid: TransactionId): Future[Unit] = runCmd("pause", id.asString).map(_ => ())
+
+    def resume(id: ContainerId)(implicit transid: TransactionId): Future[Unit] = runCmd("resume", id.asString).map(_ => ())
+
+    private def runCmd(args: String*)(implicit transid: TransactionId): Future[String] = {
+        val cmd = runcCmd ++ args
```

Comment:
Ok so logging is now explicit on process calls for docker. (Checking for myself if this is the way it already is)